core/sql/parser/sqlparser.y

```diff
@@ -24909,7 +24946,6 @@ create_table_as_token: TOK_AS
 
 /* type pElemDDL */
 table_definition_body : table_element_list
-                      | like_definition
                       | external_table_definition
 
```

Comment:
Incidentally, I did a full run of all regression tests having CREATE TABLE LIKE over the weekend to make sure I didn't miss anything. They all passed.